### DISCUSSION TITLE
quiz的分数重复提交以后一直不刷新

### DISCUSSION THREAD

**Discussion by grantwzq** [AmazingAng/WTF-Solidity#368] [Q&A] [answered]
这个问题还是没有修复吗?

**Comment by AmazingAng** [comment, marked as answer]
https://github.com/AmazingAng/WTF-Solidity/discussions/260

**Reply by grantwzq** [reply]
看过这个帖子，实际情况是依然不刷新重新提交后的quiz分数，我尝试了全部选a，quiz分数还是不变

**Reply by AmazingAng** [reply]
再仔细读一次

**Reply by grantwzq** [reply]
> 再仔细读一次

入门的quiz2提交真的有问题，就是不刷新分数，我试过全选a或b或c或d，分数都是同样的

**Reply by AmazingAng** [reply]
> > 再仔细读一次
> 
> 入门的quiz2提交真的有问题，就是不刷新分数，我试过全选a或b或c或d，分数都是同样的

成绩会记录你的历史最高成绩，如果你重新提交作业但成绩没有上升，则不会刷新成绩。

**Reply by Rjohn666** [reply]
入门第二章分数刷不出来，别的章节都全部通过了
